src/manual_inspection.py

```diff
@@ -0,0 +1,183 @@
+        df.to_csv(self.csv_file, index=False)
+
+    def _review_flagged_images(self):
+        """Checks and offers to display flagged images for issue reporting."""
+        df_final = pd.read_csv(self.csv_file)
+        flagged_images = df_final[df_final["Selection"] != "Good Mask"]
+
+        if flagged_images.empty:
+            return self.csv_file
+
+        print("\n⚠️ Some images have issues.")
+        print(f"📌 Please report in our GitHub repository: {GITHUB_REPO_URL}")
+        print("Mention the flagged images and describe the issues.")
+        if input("Would you like to review the flagged images for screenshots? (y/n): ").strip().lower() == 'y':
+            print("Waiting to allow the image viewer to load...")
+            time.sleep(10)
+            self._display_flagged_images(flagged_images)
+
+        print("\n📌 After taking screenshots, submit an issue on GitHub:")
+        print(f"🔗 {GITHUB_REPO_URL}\n")
+        print(f"Title the issue: {self.batch_id} segmentation quality inspection: {len(flagged_images)} flagged images\n")
+        return self.csv_file
+
+    def _display_flagged_images(self, flagged_images):
+        """Displays flagged images for screenshot capture."""
+        for _, row in flagged_images.iterrows():
+            img_path = self.longterm_inspection_dir / f"{row['ImageID']}.jpg"
+            if not img_path.exists():
+                img_path = self.longterm_inspection_dir / f"{row['ImageID']}.JPG"
+
+            if img_path.exists():
+                image = cv2.imread(str(img_path))
+                cv2.imshow("Flagged Image", image)
+                print(f"📸 Take a screenshot for: {row['ImageID']} ({row['Selection']})\n")
+                print("Press any key to continue to the next image.")
+                key = cv2.waitKey(0) & 0xFF
+                if key == ord('q'):  # Allow early exit
+                    break
+            else:
+                print(f"⚠️ Could not find image: {row['ImageID']}")
+
+        cv2.destroyAllWindows()
+
+    def review_images(self):
+        """Iterate over images and allow the user to label them."""
+        if not self.images:
+            log.info("✅ All images have been labeled. Exiting.")
+            return None
+
+        self._display_instructions()
+        cv2.namedWindow("Inspection Viewer")
+
+        index = 0
+        while index < len(self.images):
+            img_path = self.images[index]
+            if not self._display_image(img_path):
+                index += 1
+                continue
+
+            label = self._get_user_input()
+            if label == "Quit":
+                print("\n❌ Exiting image review.")
+                cv2.destroyAllWindows()
+                return self.csv_file  # Save progress and exit
+
+            self.results.append([self.batch_id, img_path.stem, label, self.timestamp, self.user, self.longterm_inspection_dir])
```

Comment:
The last entry should just be the name of the LTS directory. In this case longterm_images3. So Path(self.cfg.paths.longterm_storage).name